taskcluster/kinds/repackage/kind.yml

```diff
@@ -47,23 +49,29 @@ tasks:
 
     macpkg:
         description: repackage mozillavpn macpkg
-        run-on-tasks-for: [github-push]
+        run-on-tasks-for: [github-pull-request, github-push]
         fetches:
             fetch:
                - macos-cmake
-            signing:
+            from-dependency: # Updated to match dependencies in repackage.py
```

Comment:
I wouldn't suggest addressing it here, but if you wanted to, an alternative solution to this would be to make signing work for PRs. We do this in other places (notably Gecko), but providing a separate pool of production grade workers that signs with non-production certificates. Doing this would simplify this repackage logic, and improve what PR testing covers.

Reply:
I would love that actually, especially if we could make the output of PR's installable by QA and authorized developers (eg: by signing with a WWDR/Developer certificate).